python/lambda-proxy/README.md

```diff
@@ -0,0 +1,65 @@
+
+# Welcome to your CDK Python project!
```

Comment:
Please remove the generated README and add one that describes your example.

Reply:
Bump so I can merge this in